webextension/background.js

```diff
@@ -0,0 +1,13 @@
+browser.tabs.onCreated.addListener(tab => {
+  if (tab.incognito) {
+    browser.browserAction.disable(tab.id);
```

Comment:
let's add `browser.browserAction.setTitle("Containers disabled in Private Browsing Mode")` so we can show the user what's happening via the tooltip.